make data fails on fresh project

The error is

```
Usage: make_dataset.py [OPTIONS] INPUT_FILEPATH OUTPUT_FILEPATH

Error: Missing argument "input_filepath".
Makefile:30: recipe for target 'data' failed
make: *** [data] Error 2
```

and seems can be solved with [https://github.com/drivendata/cookiecutter-data-science/blob/master/%7B%7B%20cookiecutter.repo_name%20%7D%7D/Makefile#L30](Makefile#L30) changing
```
	$(PYTHON_INTERPRETER) src/data/make_dataset.py
```
to
```
	$(PYTHON_INTERPRETER) src/data/make_dataset.py data/raw data/processed
```

--- a/{{ cookiecutter.repo_name }}/Makefile
+++ b/{{ cookiecutter.repo_name }}/Makefile
@@ -27,7 +27,7 @@ requirements: test_environment
 
 ## Make Dataset
 data: requirements
-	$(PYTHON_INTERPRETER) src/data/make_dataset.py
+	$(PYTHON_INTERPRETER) src/data/make_dataset.py data/raw data/processed
 
 ## Delete all compiled Python files
 clean:

